src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/samplers/MetricsDBFileSampler.java

```diff
@@ -0,0 +1,83 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.samplers;
```

Comment:
nit: Add License information.

Reply:
Sure

Reply:
nit: Please update the year to 2020, instead of 2019.

Reply:
Sure